sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ToString.java

```diff
@@ -38,74 +37,67 @@ private ToString() {
   }
 
   /**
-   * Returns a {@code PTransform<PCollection, PCollection<String>>} which transforms each
-   * element of the input {@link PCollection} to a {@link String} using the
-   * {@link Object#toString} method.
+   * Transforms each element of the input {@link PCollection} to a {@link String} using the {@link
+   * Object#toString} method.
    */
-  public static PTransform<PCollection<?>, PCollection<String>> elements() {
-    return new SimpleToString();
+  public static Elements elements() {
```

Comment:
No reason. I thought all of the public to private changes were being done in a different PR and all at once.